assets/pulpcore-api.prep

```diff
@@ -6,6 +6,15 @@ if -n {
 foreground {
   mkdir -p /etc/pulp/certs
 }
+if -n {
+  ls /etc/pulp/certs/database_fields.symmetric.key
+}
+foreground {
+  openssl rand -base64 32 > /etc/pulp/certs/database_fields.symmetric.key
+}
+if -n {
+  ls /etc/pulp/certs/token_private_key.pem
+}
 foreground {
   openssl ecparam -genkey -name prime256v1 -noout -out /etc/pulp/certs/token_private_key.pem
 }
```

Comment:
I think, the foreground needs to wrap your if construct. See example here:

https://skarnet.org/software/execline/if.html

Reply:
Does [this if](https://github.com/pulp/pulp-oci-images/blob/latest/assets/pulpcore-api.prep#L3-L5) need to be wrapped in a foreground construct too?

Reply:
Let's say, the way it's written now,i think it runs all three subsequent commands like they were in a block.

execline is weird!

Reply:
Yea, I read through the docs and I am still not sure I understand it completely. I pushed a change that I hope is correct?